src/argosim/plot_utils.py

```diff
@@ -291,3 +292,59 @@ def plot_uv_hist(baselines, bins=20, output_folder=None):
         plt.show()
 
     return baseline_hist
+
+
+from metrics_utils import compute_eccentricity
+
+
+def plot_beam_fit_ellipse(beam, fit_result, zoom=30):
```

Comment:
Same idea for fit_result. You can set the default to `None` and in case the user doesn't give the fit result the fit is computed in the function and then plotted.